install/kubernetes/helm/istio-init/files/crd-certmanager-10.yaml

```diff
@@ -48,6 +48,28 @@ metadata:
     heritage: Tiller
     release: istio
 spec:
+  additionalPrinterColumns:
```

Comment:
You can't add this to -10 I think - need to create a crd-cermanager-11.yaml  ( some obscure reason I don't remember...)

Reply:
Moved the new CRD-s to the 'crd-certmanager-11.yaml', introduced corresponding configmap wrapper and job: 'configmap-crd-certmanager-11.yaml' and 'job-crd-certmanager-11.yaml'. Already squashed the two commit together.